scripts/utils/setup-roles.ts

```diff
@@ -78,7 +81,7 @@ export async function main(
         throw new Error("Must specify LOAN_CORE_ADDRESS in environment!");
     }
 
-    if (!admin) {
+    if (!ADMIN_ADDRESS) {
```

Comment:
Lines 94-97 below also don't make sense, it appears we:

1) Have a function that takes the `loanCore` as a `Contract` (line 48)
2) Take the address from that contract (line 75)
3) Create a _new_ instance of the contract, using the address taken from the old instance (line 95)

I'm not sure how we got here but if are already getting the contract in the function arguments, there shouldn't be any reason to do this.